### PR TITLE
ODATA-1403 - applied - Annotation of implicit entity sets

### PR DESCRIPTION
**Pull request by HeikoTheissen**
https://issues.oasis-open.org/browse/ODATA-1403

### CODE REVIEW

File: vocabularies/Org.OData.Aggregation.V1.xml [hunk 1]
```diff
@@ -136,7 +136,8 @@
         <Annotation Term="Core.Description" String="This property can be used in the `aggregate` transformation" />
       </Term>
 
-      <Term Name="CustomAggregate" Type="Edm.String" Nullable="false" AppliesTo="EntitySet ComplexType EntityContainer">
+      <Term Name="CustomAggregate" Type="Edm.String" Nullable="false" AppliesTo="EntitySet Collection ComplexType EntityContainer">
```
Comment (ralfhandl) [inline vocabularies/Org.OData.Aggregation.V1.xml:139]:
```suggestion
      <Term Name="CustomAggregate" Type="Edm.String" Nullable="false" AppliesTo="Collection ComplexType EntityContainer">
```

`Collection` includes `EntitySet`. Do you want to have it redundantly for compatibility reasons?

Comment (HeikoTheissen) [reply vocabularies/Org.OData.Aggregation.V1.xml:139]: We decided to keep both yesterday. Let's discuss next week.

**Review by ralfhandl — APPROVED** [review]

**Review by ralfhandl — APPROVED** [review]

**Review by ralfhandl — APPROVED** [review]

File: vocabularies/Org.OData.Aggregation.V1.json [hunk 2]
```diff
@@ -40,10 +40,12 @@
             "$Type": "Aggregation.ApplySupportedType",
             "$AppliesTo": [
                 "EntitySet",
+                "Collection",
                 "ComplexType",
```
Comment (ralfhandl) [inline vocabularies/Org.OData.Aggregation.V1.json:44]: Why do we need ComplexType? And why not EntityType?

File: vocabularies/Org.OData.Aggregation.V1.json [hunk 3]
```diff
@@ -120,15 +134,23 @@
...
+                }
+            ],
             "@Core.Description": "This property can be used in the `aggregate` transformation"
         },
         "CustomAggregate": {
             "$Kind": "Term",
             "$AppliesTo": [
                 "EntitySet",
+                "Collection",
                 "ComplexType",
```
Comment (ralfhandl) [inline vocabularies/Org.OData.Aggregation.V1.json:150]: Why do we need ComplexType? And why not EntityType?

File: vocabularies/Org.OData.Aggregation.V1.json (hunk 2 above)
Comment (HeikoTheissen) [reply vocabularies/Org.OData.Aggregation.V1.json:44]: @GeraldKrause, can you answer this?

**Comment by mikepizzo** [comment]
Note: We decided to defer resolution of this issue until we decide our intentions for representing the existing non-aggregation navigation property restrictions.

**Comment by HeikoTheissen** [comment]
> representing the existing non-aggregation navigation property restrictions.

But see [this comment](https://issues.oasis-open.org/browse/ODATA-1403?focusedCommentId=78937&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-78937).

**Pull request merged**